lighthouse-core/lib/stack-packs.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const stackPacks = require('@lighthouse/stack-packs');
+
+const stackPacksToInclude = [{
+  packId: 'wordpress',
+  npmOrIconName: 'wordpress',
```

Comment:
I probably missed this discussion, but what's an icon name in this context? It seems different than the `icon` also included with these stack packs...

Reply:
> I probably missed this discussion,

ya there was some.

wordpress isn't on npm.
everything we care about either is on npm or has a unique ID provided by the js-lib-detector `icon` field.

so.............. that's what this is.

it could be `detectedId` just as easily. and that probably makes more sense.

Reply:
> it could be `detectedId` just as easily. and that probably makes more sense.

`detectedId` sounds good to me. Definitely confusing with `icon` and if you don't know anything about `js-library-detector` output